src/Event/LocalFileStorageListener.php

```diff
@@ -94,12 +94,8 @@ public function afterSave(Event $event) {
 			try {
 				$filename = $this->buildFileName($table, $entity);
 				$entity->path = $this->buildPath($table, $entity);
-
 				$Storage->write($entity['path'] . $filename, file_get_contents($entity['file']['tmp_name']), true);
-				$table->save($entity, array(
-					'validate' => false,
-					'callbacks' => false
-				));
```

Comment:
Are you sure removing this here has no side effects? This will trigger this callbacks again and probably cause a loop.